project/Dependencies.scala

```diff
@@ -23,8 +22,7 @@ object Dependencies {
   private val fs2Test = "co.fs2" %% "fs2-io" % fs2V % "test"
 
   private val catsDependencies = List(
-    "org.typelevel" %% "cats-core" % catsV,
-    "com.github.benhutchison" %% "mouse" % "0.9"
+    "org.typelevel" %% "cats-core" % catsV
   ) map (_
     /*
     Exclude test framework cats-laws and its transitive dependency scalacheck.
```

Comment:
While you're here: Now that we're only using `cats-core`, remove these exclusions for `cats`.